lib/internal/HtmlLinkParser.js

```diff
@@ -339,7 +339,7 @@ function reset(instance)
 	_.linkElementStack.length = 0;
 	
 	// Per-link variables
-	_.links.length = 0;
+	_.links = [];
```

Comment:
Added a comment. I'm pretty sure the reset needs to happen since the `_.links` gets set to false sometimes. Feels safest to ensure it's an array before accessing it.